cli/dcoscli/marathon/main.py

```diff
@@ -781,6 +786,27 @@ def _task_list(app_id, json_):
     return 0
 
 
+def _task_stop(task_id, wipe):
+    """Stop a Marathon task
+
+    :param task_id: the id of the task
+    :type task_id: str
+    :param wipe: whether to wipe persistent data and unreserve resources
+    :type wipe: bool
+    :returns: process return code
+    :rtype: int
+    """
+
+    client = marathon.create_client()
+    task = client.stop_task(task_id, wipe)
+
+    if task is None:
+        raise DCOSException("Task '{}' does not exist".format(task_id))
+
+    emitter.publish(task)
```

Comment:
Sure, I've added happy path tests for `stop` and `stop --wipe` as well as a test for each case that verifies failure when sending an unknown task_id.

The return value on the happy path is a list of the stopped tasks.